### DISCUSSION TITLE
Allow final rendered video to be downloaded 🎥

### DISCUSSION THREAD

**Discussion by HamedMP** [Just-Moh-it/Remotion-Template-NextJS#8] [Ideas]
Hi! 

Thanks for the template, life saver! 

Have all the good stuff together, everything from Next to Lambda! 🎉 

One idea: I'm just playing with the editor. After pressing Render, and finishing, I didn't see any URL to the final output. 

Checked the `useRender` hook but it returns only the start function and the status. How one can get the URL of the final video?

**Comment by Just-Moh-it** [comment]
Hey Hamed, I'll make it a priority for this weekend!

(Moved to issue #9)
